views/partials/entity-details.html

```diff
@@ -14,10 +14,12 @@
 
 {% extends "layouts/two-columns.html" %}
 
-{% block banner %}
-  {% block title %}<h1><small>{{ type | capitalize }}:</small>{{ title }}</h1>{% endblock %}
-  {% block subtitle %}<p>{{ subtitle }}</p>{% endblock %}
-  {% block bannerFooter %}{% endblock %}
+{% block title %}<h1><small>{{ type | capitalize }}:</small>{{ title }}</h1>{% endblock %}
+{% block subtitle %}<p>{{ subtitle }}</p>{% endblock %}
+{% block breadcrumbs %}
+  <li><a href="/">Home</a></li>
+  <li><a href="/search">Search</a></li>
+  <li><a>{{ title | truncate(35) }}</a></li>
```

Comment:
I'm not sure as I haven't run the code, but you might be missing a `{% block banner %}` and a `{% block bannerFooter %}{% endblock %}` here?